plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/controller/ChatController.kt

```diff
@@ -116,13 +123,31 @@ class ChatController private constructor(
     }
 
     override suspend fun processPromptChatMessage(message: IncomingCwcMessage.ChatPrompt) {
+        var prompt = message.chatMessage
+        var queryResult: List<RelevantDocument> = emptyList()
+        val triggerId = UUID.randomUUID().toString()
+        var shouldAddIndexInProgressMessage: Boolean = false
+        if (prompt.contains("@workspace")) {
+            if (CodeWhispererSettings.getInstance().isProjectContextEnabled()) {
+                val projectContextController = ProjectContextController.getInstance(context.project)
+                prompt = prompt.replace("@workspace", "")
+                queryResult = projectContextController.query(prompt)
+                if (!projectContextController.getProjectContextIndexComplete()) shouldAddIndexInProgressMessage = true
+                logger.info("project context relevant document count: ${queryResult.size}")
```

Comment:
nit: should use lambda version instead
```suggestion
                logger.info { "project context relevant document count: ${queryResult.size}" }
```